test/e2e/common/pod_resize.go

```diff
@@ -425,7 +727,7 @@ var _ = ginkgo.Describe("[sig-node] PodInPlaceResize", func() {
 			}
 			rPod, rErr := waitPodStatusResourcesEqualSpecResources()
 			framework.ExpectNoError(rErr, "failed to get pod")
-			verifyPodResources(rPod, tc.expected)
```

Comment:
why did we remove this?

Reply:
I thought it already ran before the sync. Is there a need to run it twice?